pkg/ccl/backupccl/backup_test.go

```diff
@@ -1637,6 +1638,63 @@ func TestRestoreRetryProcErr(t *testing.T) {
 	})
 }
 
+// TestBackupJobRetryReset tests that the job level retry counter
+// resets after the backup made progresses. To do so, the test does the following:
+// 1. Intercept the backup job after a backup chunk completes and returns a retryable
+// error until the retry reached desired count.
+// 2. Added a new testing know BeforeBackupResetRetry to simulate backup job has made
```

Comment:
there are few typos in this sentence. please do reread comments before submitting a PR for review.